### DISCUSSION TITLE
How to sort methods?

### DISCUSSION THREAD

**Discussion by Pedrexus** [astral-sh/ruff#3506] [Q&A] [answered]
Hello, 

I'm recently migrated my linter/formatter to ruff, but before I used a flake8 plugin that would sort methods, functions, properties in alphabetically. Is there any way to achieve this functionality in ruff?

In any case, thank you for this amazing lib. The speedups are impressive!

**Comment by JonathanPlasse** [comment, marked as answer]
- There is a proposition similar to this (c.f. #2425)
